example/CountlyRNExample/App.tsx

```diff
@@ -1,1000 +1,55 @@
-                    <CountlyButton onPress={this.getRemoteConfigValueForKeyBoolean} title="Boolean Test" color="#00b5ad" />
-                    <CountlyButton onPress={this.getRemoteConfigValueForKeyFloat} title="Float Test" color="#00b5ad" />
-                    <CountlyButton onPress={this.getRemoteConfigValueForKeyInteger} title="Integer Test" color="#00b5ad" />
-                    <CountlyButton onPress={this.getRemoteConfigValueForKeyString} title="String Test" color="#00b5ad" />
-                    <CountlyButton onPress={this.getRemoteConfigValueForKeyJSON} title="JSON Test" color="#00b5ad" />
-                    <CountlyButton onPress={this.remoteConfigClearValues} title="Clear remote config cache" color="#00b5ad" />
-                    <Text style={[{ textAlign: 'center' }]}>Remote Config End</Text>
-                    <Text style={[{ textAlign: 'center' }]}>.</Text>
-                    <Text style={[{ textAlign: 'center' }]}>Crash Event start</Text>
-                    <CountlyButton onPress={this.addCrashLog} title="Add Crash Log" color="#00b5ad" />
-                    <Text style={[{ textAlign: 'center' }]}>Crash Event End</Text>
-                    <Text style={[{ textAlign: 'center' }]}>.</Text>
-                    <Text style={[{ textAlign: 'center' }]}>APM Example Start</Text>
-                    <CountlyButton onPress={this.startTrace} title="Start Trace" color="#1b1c1d" lightText={true} />
-                    <CountlyButton onPress={this.endTrace} title="End Trace" color="#1b1c1d" lightText={true} />
-                    <CountlyButton onPress={this.recordNetworkTraceSuccess} title="End Network Request Success" color="#1b1c1d" lightText={true} />
-                    <CountlyButton onPress={this.recordNetworkTraceFailure} title="End Network Request Failure" color="#1b1c1d" lightText={true} />
-                    <CountlyButton onPress={this.recordDirectAttribution} title="Record Direct Attribution" color="#1b1c1d" lightText={true} />
-                    <CountlyButton onPress={this.recordIndirectAttribution} title="Record Indirect Attribution" color="#1b1c1d" lightText={true} />
-                    <Text style={[{ textAlign: 'center' }]}>APM Example Start</Text>
-                    <Text style={[{ textAlign: 'center' }]}>.</Text>
-                    {/*
-            <CountlyButton onPress = { this.testCrash } title = "Test Native Crash" color = "crimson"/>
-            */}
-                </ScrollView>
-            </SafeAreaView>
+            <NavigationContainer>
+                <Stack.Navigator initialRouteName={navigationName.Home}>
+                    <Stack.Screen
+                        name={navigationName.Home}
+                        component={HomeScreen}
+                        options={{
+                            headerTitle: (props) => (
+                                <Image
+                                    source={require('./asset/countly-logo.png')}
```

Comment:
consider adding logo path to constants file

Reply:
again later pr